src/metadata/definitions/directive-metadata.ts

```diff
@@ -0,0 +1,14 @@
+export interface DirectiveMetadata {
+  nameOrSDL: string;
```

Comment:
We need a better name than `sdl`... `directiveDefString`? `directiveDefinition`? `directiveTypeDef`? Ideas? 😄

Reply:
`nameOrDefinition`?

Reply:
`nameOrDefinition` it is.  Or I can be explicit and have `name?: string;` and `definition?: string`

Reply:
I’d go with `nameOrDefinition` since it’s not intuitive enough which one gets excluded if you go with two fields. Does `name` overrides `definition` or the other way around? Will there be a scenario where you have both?

Reply:
I’d go with `nameOrDefinition` since it’s not intuitive enough which one gets excluded if you go with two fields. Does `name` overrides `definition` or the other way around? Will there be a scenario where you have both?

Reply:
You would have to do the parse phase in decorators body, which is not a great separation tho.